domains/transfer-agreements/TransferAgreements.API/API.UnitTests/Controllers/TransferAgreementsControllerTests.cs

```diff
@@ -39,20 +39,29 @@ public TransferAgreementsControllerTests()
 
 
     [Fact]
-    public async Task Create_ShouldCallServiceOnce()
+    public async Task Create_ShouldCallRepositoryOnce()
```

Comment:
I would say these are covered via integration tests? Don't delete these now that we have them :)